textFile.txt

```diff
@@ -0,0 +1,11 @@
+The example above will fetch the first 10 lines of the file you have selected. 
+This means streams can be very useful for handling large files or even slow resources 
+like network resources.
+
+The amount of functionality in the Enum and Stream modules can be daunting at first, 
+but you will get familiar with them case by case. In particular, focus on the Enum 
+module first and only move to Stream for the particular scenarios where laziness is 
+required, to either deal with slow resources or large, possibly infinite, collections.
+
+Next, we’ll look at a feature central to Elixir, Processes, which allows us to write 
+concurrent, parallel and distributed programs in an easy and understandable way.
```

Comment:
What is this file about?

Reply:
Its a text file to test some features about file management in Elixir, like read line per line, write new lines and how is this stored.